cli/cmd/start/start.go

```diff
@@ -204,18 +206,34 @@ func StartCmd(ch *cmdutil.Helper) *cobra.Command {
 	return startCmd
 }
 
+// a smaller subset of relevant parts of rill.yaml
+type rillYAML struct {
+	IgnorePaths []string `yaml:"ignore_paths"`
+}
+
 func countFilesInDirectory(path string) (int, error) {
 	var fileCount int
 
 	if path == "" {
 		path = "."
 	}
 
-	err := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+	var ignorePaths []string
+	// Read rill.yaml and get `ignore_paths`
+	rawYaml, err := os.ReadFile(filepath.Join(path, "/rill.yaml"))
+	if err == nil {
+		yml := &rillYAML{}
+		err = yaml.Unmarshal(rawYaml, yml)
+		if err == nil {
+			ignorePaths = yml.IgnorePaths
+		}
+	}
+
+	err = filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
```

Comment:
Can you change this to `filepath.WalkDir` and return `filepath.SkipDir` if the path is ignored? So that it doesn't spend time traversing ignored directories.

Reply:
Good point, updated.